### PR TITLE
Support quick access to top nav items with Alt 1–9 keyboard shortcuts

### LINKED ISSUE #1286: Support quick access to top nav items with ⌘1–⌘9 keyboard shortcuts
This feature is about increasing speed and productivity for Bisq power users (especially @bisq-network/arbitrators!) by reducing the need to use the mouse for navigation.

This, like another recent bounty we did in #1144, is just one of many such keyboard shortcut implementations that can add up to better productivity and enjoyment when using Bisq. Another would be supporting alt+letter key combinations for navigating to submenu items (but that would be managed as a separate issue; I'm just mentioning it here for context).

image

Of course, it wouldn't be ⌘ keys on Windows and Linux, it would be whatever is idiomatic there for this kind of navigation.

Note that while there are usually only 8 top nav items, the DAO tab can be enabled with ⌘D, making a 9th.

### PR DESCRIPTION
**Pull request by blabno**
Fixes #1286

### CODE REVIEW

**Comment by blabno** [comment]
Supports left Alt on linux. I need to obtain Mac to test that rounted sharp key.

**Comment by cbeams** [comment]
I just quickly tested on mac, none of the keys work.

**Comment by cbeams** [comment]
Also, @blabno, it's probably best to implement this in a way that follows the existing approach for supporting keyboard shortcuts, which you can find at https://github.com/bisq-network/exchange/blob/master/gui/src/main/java/io/bisq/gui/app/BisqApp.java#L249-L284. If you follow that convention, these keyboard shortcuts should "just work" across all platforms, or at the very least, they will work as well as other, existing, keyboard shortcuts do across all platforms.

For example, line https://github.com/bisq-network/exchange/blob/master/gui/src/main/java/io/bisq/gui/app/BisqApp.java#L261, is where the `CMD-J` shortcut is configured, which opens the emergency wallet tool.

**Comment by blabno** [comment]
@cbeams I've pushed new commit using suggested approach. You can give it a try.

File: gui/src/main/java/io/bisq/gui/main/MainView.java [hunk 1]
```diff
@@ -27,7 +27,11 @@
 import io.bisq.core.app.BisqEnvironment;
 import io.bisq.core.exceptions.BisqException;
 import io.bisq.gui.Navigation;
-import io.bisq.gui.common.view.*;
+import io.bisq.gui.common.view.CachingViewLoader;
+import io.bisq.gui.common.view.FxmlView;
+import io.bisq.gui.common.view.InitializableView;
+import io.bisq.gui.common.view.View;
+import io.bisq.gui.common.view.ViewLoader;
```
Comment (cbeams) [inline gui/src/main/java/io/bisq/gui/main/MainView.java]: Please don't expand these imports or the others that have been expanded below. The built-in IDEA settings should handle this correctly when running `Optimize Imports`. You may not be sync'd up correctly with those settings, or perhaps you're using some other IDE, in which case you'll need to take greater care to get it right manually.

In any case, please keep PRs atomic and avoid mixing in unrelated changes.

**Review by cbeams — CHANGES_REQUESTED** [review]
NACK. The substance of the PR looks good, and I've tested it locally on my Mac, where everything works as expected. Just need to clean up the imports and we're good.

File: gui/src/main/java/io/bisq/gui/main/MainView.java (hunk 1 above)
Comment (blabno) [reply gui/src/main/java/io/bisq/gui/main/MainView.java]: Sorry, I didn't see `.gitignore` ignores `.idea` but actually keeps code style.
I've optimized imports using Bisq code style.

**Review by cbeams — APPROVED** [review]

**Comment by cbeams** [comment]
ACK, thanks @blabno!

Note that the next compensation request voting period starts tomorrow, and you have until end of day today to submit a request for this work if you'd like to. Otherwise you can always submit for a subsequent voting period, no problem.

Instructions are at https://github.com/bisq-network/docs/blob/master/dao/phase-zero.adoc#submit-a-compensation-request.

With regard to how much you should ask for, please gauge that based on how much value you believe this change adds to Bisq, the Bisq network, and its users. Not "how much time you spent on it". You can look at other compensation requests to get a gauge of things, and of course, feel free to ask around in Slack or elsewhere to see if the amount you're requesting seems reasonable to others.

**Pull request merged**